methods.py

```diff
@@ -1044,3 +1044,32 @@ def non_serializable(obj):
 
     with open(".scons_env.json", "w") as f:
         dump(env.Dictionary(), f, indent=4, default=non_serializable)
+
+
+def get_mvk_sdk_path():
```

Comment:
This could likely be defined in `platform/osx/detect.py` itself if it's not meant to be shared with other parts of the buildsystem.

Reply:
Moved. iOS does not use SDK since it's building static library only, and it's irrelevant for other platforms.